portal-impl/src/portal.properties

```diff
@@ -6928,6 +6933,21 @@
     feature.flag.LPS-180155.description=Enable dataset display for publications to access table configuration and sort options in table headers when viewing ongoing, scheduled, or published publications.
 
     #
+
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASES__MINUS__NUMBER1__NUMBER8__NUMBER4__NUMBER4__NUMBER0__NUMBER4__PERIOD_DESCRIPTION
+    #
+    feature.flag.LPS-184404.description=Enables full width for Control Menu, Management Bar, and Navigation Bar
```

Comment:
Should be "Fluid Layout provides a more clear UI for large screen sizes, specifically for DXP admin pages." This is defined in the ticket.